rdmd.d

```diff
@@ -154,6 +156,12 @@ int main(string[] args)
             "man", { man(); bailout = true; },
             "tmpdir", &userTempDir,
             "o", &dashOh);
+    } catch (Exception e)
```

Comment:
Braces on a new line:
https://dlang.org/dstyle.html#phobos_brackets

We usually have CI check this on other repos, but it's not set up here.

Reply:
Do we have a way to auto-format it?

Reply:
There's Dfmt, but I don't know how far it can get the code into a style that would pass the usual checks.